project/UI/Playground/Playground.swift

```diff
@@ -2,100 +2,94 @@
 
-enum PlaygroundCommand: String {
-    case name
-    case output
-    case playground
-    case platform
+public struct PlaygroundCommand: ConsoleCommand {
+    public static var commandName: String = "nef-playground"
+    public static var configuration = CommandConfiguration(commandName: commandName,
+                                                           abstract: "Build a playground compatible with external frameworks")
+
+    public init() {}
     
-    case bowVersion = "bow-version"
-    case bowBranch  = "bow-branch"
-    case bowCommit  = "bow-commit"
-    case podfile
-    case cartfile
-}
+    @ArgumentParser.Option(help: "Path where nef Playground will be generated")
+    var output: String
 
+    @ArgumentParser.Option(default: "BowPlayground", help: ArgumentHelp("Specify the name for the nef Playground", valueName: "playground name"))
+    var name: String
 
-private func nefPlayground<A>(xcodePlayground: URL, name: String, output: URL, platform: Platform, dependencies: PlaygroundDependencies) -> EnvIO<CLIKit.Console, CLIKit.Console.Error, A> {
-    EnvIO { console in
-        nef.Playground.nef(xcodePlayground: xcodePlayground, name: name, output: output, platform: platform, dependencies: dependencies)
-                      .provide(console)^
-                      .mapError { _ in .render() }
-                      .foldM({ e in console.exit(failure: "building nef Playground from Xcode Playground '\(xcodePlayground.path)'. \(e)") },
-                             { _ in console.exit(success: "nef Playground created successfully in '\(output.path)'")                       })^
+    @ArgumentParser.Option(default: .ios, help: "set the target to `ios` or `macos`")
+    var platform: Platform
+    
+    @ArgumentParser.Option(default: ArgumentEmpty, help: "Xcode Playground to be transformed into nef Playground")
+    var playground: String
+    
+    @ArgumentParser.Option(default: ArgumentEmpty, help: "Path to Podfile with your own dependencies")
+    var podfile: String
+    
+    @ArgumentParser.Option(default: ArgumentEmpty, help: "Path to Cartfile with your own dependencies")
+    var cartfile: String
+    
+    @ArgumentParser.Option(default: ArgumentEmpty, help: "Specify the version of Bow")
+    var bowVersion: String
+    
+    @ArgumentParser.Option(default: ArgumentEmpty, help: "Specify the branch of Bow")
+    var bowBranch: String
+    
+    @ArgumentParser.Option(default: ArgumentEmpty, help: "Specify the commit of Bow")
+    var bowCommit: String
+    
+    var outputURL: URL { URL(fileURLWithPath: output.trimmingEmptyCharacters.expandingTildeInPath) }
+    var playgroundURL: URL? { playground == ArgumentEmpty ? nil : URL(fileURLWithPath: playground.trimmingEmptyCharacters.expandingTildeInPath) }
+    var podfileURL: URL  { URL(fileURLWithPath: podfile.trimmingEmptyCharacters.expandingTildeInPath) }
+    var cartfileURL: URL { URL(fileURLWithPath: cartfile.trimmingEmptyCharacters.expandingTildeInPath) }
```

Comment:
Same thing I mentioned above about `trimmingEmptyCharacters.expandingTildeInPath`.

Reply:
Dont worry, I will propagate it in the whole UI module ^^ - Thanx